benchmarks/docker/docker_build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Builds a docker image nighthawk-dev:latest containing the stripped binaries
+# based on a pre-build bazel-bin directory (with "-c opt" set).
+
+# Stop on errors.
+set -e
+
+# NOTE: explicit no -x for verbose commands. Because this is run in CI, doing so may result in
+# publishing sensitive information into public CI logs if someone makes a change in a
+# consuming script that is off guard.
+
+DOCKER_NAME="benchmark"
+DOCKER_IMAGE_PREFIX="oschaaf/${DOCKER_NAME}"
+BAZEL_BIN="$(bazel info bazel-bin)"
+WORKSPACE="$(bazel info workspace)"
+bazel build //benchmarks:benchmarks
```

Comment:
Why not use `BAZEL_BIN` here?

Reply:
I'm not sure what you mean here, could you elaborate?

Reply:
I think I was confused by `BAZEL_BIN` above. This is the `bazel-bin` dir, not the bazel binary..